test/vaulttracker/VaultTracker.sol

```diff
@@ -18,6 +18,7 @@ contract VaultTracker {
   address public immutable swivel;
   uint256 public immutable maturity;
   uint256 public maturityRate;
+  uint256 private mathConstant = 1e26;
```

Comment:
this is maybe the worst name ever... 

is a constant better that a literal for gas or something? really? i need proof on this one.

Reply:
Lmao had no idea what to call this, in multiple tickets in the audit people called it our "magic number". No gas savings, but was recommended to ensure code is clean / always using the same magic number

Reply:
there is no name on earth which is clearer than 1e26. it's a fuckin type literal